src/FSharpPlus.AspNetCore.Suave/Library.fs

```diff
@@ -1,14 +1,14 @@
 ﻿module FSharpPlus.AspNetCore.Suave
 open FSharpPlus
 open FSharpPlus.Data
+open FSharpPlus.Control
```

Comment:
I don't think you need to open `FSharpPlus.Control`

Reply:
Yes, that's true